src/data/auroradb.ts

```diff
@@ -0,0 +1,255 @@
+import AWS from 'aws-sdk';
+import dateFormat from 'dateformat';
+import { createPool } from 'mysql2/promise';
+
+export interface ServiceType {
+    lineName: string;
+    startDate: string;
+    description: string;
+}
+
+export interface JourneyPattern {
+    startPoint: {
+        Id: string;
+        Display: string;
+    };
+    endPoint: {
+        Id: string;
+        Display: string;
+    };
+    stopList: string[];
+}
+
+export interface QueryData {
+    operatorShortName: string;
+    serviceDescription: string;
+    lineName: string;
+    fromAtcoCode: string;
+    toAtcoCode: string;
+    fromCommonName: string;
+    toCommonName: string;
+    journeyPatternId: string;
+    order: string;
+}
+export interface RawJourneyPatternStops {
+    OrderedStopPoints: {
+        StopPointRef: string;
+        CommonName: string;
+    }[];
+}
+
+export interface RawJourneyPattern {
+    JourneyPattern: RawJourneyPatternStops[];
+}
+
+interface NaptanInfo {
+    commonName: string;
+    naptanCode: string;
+    atcoCode: string;
+    nptgLocalityCode: string;
+    localityName: string;
+    parentLocalityName: string;
+    indicator: string;
+    street: string;
+}
+
+interface NaptanAtcoCodes {
+    naptanCode: string;
+    atcoCode: string;
+}
+
+export interface Stop {
+    stopName: string;
+    naptanCode: string;
+    atcoCode: string;
+    localityCode: string;
+    localityName: string;
+    parentLocalityName: string;
+    indicator?: string;
+    street?: string;
+    qualifierName?: string;
+}
+
+export interface StopIdentifiers {
+    naptanCode: string | null;
+    atcoCode: string;
+}
+
+export interface Service {
+    serviceDescription: string;
+    operatorShortName: string;
+    journeyPatterns: JourneyPattern[];
+}
+
+export interface RawService {
+    serviceDescription: string;
+    operatorShortName: string;
+    journeyPatterns: RawJourneyPattern[];
+}
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
We should be able to put the return type here as `Pool` imported from `mysql2/promise` and get rid of this ignore